azure_monitor/src/azure_monitor/export/__init__.py

```diff
@@ -204,3 +209,105 @@ def get_metrics_export_result(result: ExportResult) -> MetricsExportResult:
     ):
         return MetricsExportResult.FAILURE
     return None
+
+
+# pylint: disable=too-many-statements
+# pylint: disable=too-many-branches
+def convert_span_to_envelope(span: Span) -> protocol.Envelope:
```

Comment:
Why move this here?

Reply:
Need this conversion in LiveMetrics processing, could move to a more appropriate location like utility or something, any suggestion?

Reply:
Can we leave this in trace and have LiveMetrics just import from trace module?